src/Workspaces/Core/Portable/CaseCorrection/AbstractCaseCorrectionService.cs

```diff
@@ -36,7 +38,7 @@ public SyntaxNode CaseCorrect(SyntaxNode root, ImmutableArray<TextSpan> spans, W
             return CaseCorrect(null, root, spans, workspace, cancellationToken);
         }
 
-        private SyntaxNode CaseCorrect(SemanticModel semanticModel, SyntaxNode root, ImmutableArray<TextSpan> spans, Workspace workspace, CancellationToken cancellationToken)
+        private SyntaxNode CaseCorrect(SemanticModel? semanticModel, SyntaxNode root, ImmutableArray<TextSpan> spans, Workspace workspace, CancellationToken cancellationToken)
```

Comment:
Why is semanticModel nullable? The case where this is a language that doesn't have them?

Reply:
I'm not sure, but the preceding method passes null for it.

Reply:
Ah, maybe for syntax-only case correction (like correcting VB syntax) vs. the semantic corrections too. Well, good find.